core/src/test/scala/integration/kafka/api/SaslMultiMechanismConsumerTest.scala

```diff
@@ -41,8 +43,9 @@ class SaslMultiMechanismConsumerTest extends BaseConsumerTest with SaslSetup {
     closeSasl()
   }
 
-  @Test
-  def testMultipleBrokerMechanisms(): Unit = {
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumName)
+  @ValueSource(strings = Array("zk", "kraft", "kraft+kip848"))
+  def testMultipleBrokerMechanisms(quorum: String): Unit = {
```

Comment:
The test fails for kraft and kraft+kip848 with timeout error in createTopics in setup. The server repeatedly runs handleFetchRequest and causes the timeout. I don't know what could cause the loop...